erpnext/payroll/doctype/additional_salary/additional_salary.json

```
@@ -171,11 +177,34 @@
    "print_hide": 1,
    "read_only": 1,
    "reqd": 1
+  },
+  {
+   "fieldname": "employee_details_section",
+   "fieldtype": "Section Break",
+   "label": "Employee Details"
+  },
+  {
+   "fieldname": "salary_component_and_amount_section",
+   "fieldtype": "Section Break",
+   "label": "Salary Component And Amount"
```

```suggestion
   "fieldname": "salary_details_section",
   "fieldtype": "Section Break",
   "label": "Salary Details"
```
Since this section also covers dates and recurring property.